doc/topics/releases/3005.rst

```diff
@@ -5,3 +5,117 @@ Salt 3005 Release Notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+
+Python 3.5 and 3.6 Deprecation
+------------------------------
+
+This will be the last release we will support Python versions 3.5 and 3.6.
+In Salt release 3006 we will only support Python versions 3.7 and higher.
+Going forward our policy will be to align with Python's supported versions.
+
+
+New packages available
+----------------------
+
+With the release of Salt 3005, we are pleased to announce the
+new packages using Tiamat and pyinstaller are now out of beta
+and ready for production. These new packages are self-contained
+binaries of Salt, which includes the version of Python needed by Salt
+and the required dependencies of Salt. The Tiamat packages allow users
+to use Salt without Python being installed on the box.
+
+Any new OS platforms we support going forward from version 3005 will only
+include Tiamat packages. For this release, this includes Centos 8 Stream,
+Centos 9 Stream and Ubuntu 22.04.
+
+On the day of the Phosphorus release, the instructions on how to install the
+new Tiamat packages will be available on https://repo.saltproject.io for each
+platform. If you want to test out the packages today you can install them
+from https://repo.saltproject.io/salt-dev/py3/ using the correct directory
+for your platform. If you find any issues with the packages, please open an
+issue on this repo: https://gitlab.com/saltstack/open/salt-pkg
+
+
+Non-Tiamat packaging support
+----------------------------
+The non-Tiamat packaging system previously used for Salt will also be provided
+for platforms supported in previous Salt versions. The non-Tiamat packaging will
+only be avilable for two more releases. Please see the :ref:`release-3006` notes for more
+details on the availablility of Non-Tiamat packaging for the 3006 release.
+The 3007 release and those going forward will only provide the Tiamat packages.
+
+Platform Package Support
```

Comment:
Minor (non-blocking): "Platform package support" should be in sentence case, not title case. Per Google Developer's Style Guide: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings